index.html

```diff
@@ -1696,7 +1697,6 @@ <h2>The <dfn>NFCPushTarget</dfn> enum</h2>
           USVString url = "";
           NFCRecordType recordType;
           USVString mediaType = "";
-          NFCAccept accept = "web-nfc-only";
```

Comment:
Do you mean that if reader options is specified (recordType or mediaType is present), then by default it's web-nfc-only and otherwise not?

Shouldn't this be explicitly mentioned in the spec?

Reply:
If you add any filter (URL) it will require the NFC record

Reply:
OK, but I think then this needs to be stated in prose in the spec and if so, it should be in this PR. Could you add a clarifying statement?

Reply:
I could add a note, but it is clear from the algorithm. You want me to add a comment to the commit message?

Reply:
Yes, it is implied by the algorithm, but I wonder people would really realize the behavior here. Adding a note would be good IMHO, at least for now.